src/app/shell/personal-cabinet/provider/create-provider/create-contacts-form/create-contacts-form.component.scss

```diff
@@ -1,2 +1,13 @@
 @import "src/app/shared/styles/create-form.scss";
-@import "src/app/shared/styles/validation-form.scss";
\ No newline at end of file
+@import "src/app/shared/styles/validation-form.scss";
+
+.checkbox {
```

Comment:
Please, get rid of these styles. We already have styles in the shared scss files. You need to add class " class="step-check-box". If you want to change it, please do it in the shared one, so we will have the same for all forms. 
![image](https://user-images.githubusercontent.com/27493959/121986694-a5cb7280-cd9f-11eb-9fc8-97739a7943a9.png)